tapir/coop/templates/coop/log/create_payment_log_entry.html

```diff
@@ -1,2 +1,2 @@
 {% load i18n %}
-{% blocktranslate with amount=entry.amount payment_date=entry.payment_date %}New Payment: {{amount}} € on {{payment_date}}{% endblocktranslate %}
\ No newline at end of file
+{% blocktranslate with amount=entry.amount payment_date=entry.payment_date paying_member=entry.paying_member.get_info.get_display_name %}New Payment: {{amount}} € on {{payment_date}} paid by {{paying_member}}{% endblocktranslate %}
```

Comment:
On the payment list page I anonymised the payer's name for people that are not in the member office. I think it's better because people who are on the matching-program list don't choose who the money goes to. See 
https://github.com/SuperCoopBerlin/tapir/blob/92514645104b2331509217ed7a7fdb16b91bcfa8/tapir/coop/views/incoming_payments.py#L42-L51